### ISSUE TITLE
Duplicate UUID's if same user is logged into client

### ISSUE THREAD

**Issue by MosDefAssassin** [byt3bl33d3r/gcat#4]
Currently the implant.py script is set to use the environment variable USERNAME. I would change this to use COMPUTERNAME which will be more random or something that could be more random. Case in point if I own two machines that the user account Administrator is logged into, they will get the same UUID. 

#generates a unique uuid 
uniqueid = str(uuid.uuid5(uuid.NAMESPACE_OID, os.environ['COMPUTERNAME']))

I changed it in my copy and it seems to have fixed the issue.
Love this project by the way... Genius. ;)

**Comment by byt3bl33d3r** [comment]
@MosDefAssassin hi, sorry for the late response.
The *first* part of the UUID string should be different on two machines even if the name of the user account is the same.
What OS'es are you using?

**Comment by MosDefAssassin** [comment]
The two test machines were both running windows 7. They were also virtual
machines. I was logged in with the same user name on both and noticed the
problem. I haven't had the issue since I applied my fix. I went back and
looked through the gmail account I was using and could see that there was
definitely two of the same UUIDs. I also tested it on a windows 2008 server
which reported in with a different UUID around the same time frame so I
know the problem was occurring.
